tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_deposit.py

```diff
@@ -0,0 +1,40 @@
+from eth2spec.test.context import (
+    spec_state_test,
+    always_bls,
+    with_altair_and_later,
+    with_phases,
+)
+from eth2spec.test.helpers.constants import (
+    ALTAIR,
+)
+
+
+from eth2spec.test.helpers.deposits import (
+    run_deposit_processing_with_specific_fork_version,
+)
+
+
+@with_phases([ALTAIR])
```

Comment:
can this be "altair and later"?

Reply:
No, this is the control group of `bellatrix/block_processing/test_process_deposit.py::test_ineffective_deposit_with_previous_fork_version`. Altair's previous fork is exactly the correct `GENESIS_FORK_VERSION` while others' are not.

Added more comments (https://github.com/ethereum/consensus-specs/pull/3143/commits/34907d23758dd8b7394226514bfa66f8b6a6b1cb) and `current_fork_version` test case (https://github.com/ethereum/consensus-specs/pull/3143/commits/e5709a55b6f998e424fe454684ffe36542da06a5).